src/components/input/SoftwareBuildDownloadButton.tsx

```diff
@@ -0,0 +1,135 @@
+import { Menu, Transition } from "@headlessui/react";
+import clsx from "clsx";
+import { Fragment, useState } from "react";
+
+import CloneIcon from "@/assets/icons/fontawesome/clone-icon.svg";
+import ChevronDownIcon from "@/assets/icons/heroicons/chevron-down.svg";
+import DocumentDownloadIcon from "@/assets/icons/heroicons/document-download.svg";
+import type { BuildDownload } from "@/lib/service/types";
+import { getVersionBuildDownloadURL } from "@/lib/service/v2";
+
+interface SoftwareBuildDownloadButtonProps {
+  project: string;
+  version: string;
+  build: number;
+  downloads: Record<string, BuildDownload>;
+  stable: boolean;
+}
+
+const SoftwareBuildDownloadButton = ({
```

Comment:
Would it be possible to somehow combine this together with `SoftwareDownloadButton`? Seems like we have too much duplicated code here.